testutils/src/main/scala/za/co/absa/CmdConfig.scala

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2018 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.standardization
+
+import scopt.OptionParser
+
+/**
+  * This is a class for configuration provided by the command line parameters
+  *
+  * Note: scope requires all fields to have default values.
+  *       Even if a field is mandatory it needs a default value.
+  */
+case class CmdConfig(rawFormat: String = "xml",
+                     rowTag: Option[String] = None,
+                     csvDelimiter: Option[String] = None,
+                     csvHeader: Option[Boolean] = Some(false),
+                     fixedWidthTrimValues: Option[Boolean] = Some(false),
+                     stdPath: String = "",
+                     refPath: String = "",
+                     outPath: String = "")
+
+object CmdConfig {
+
+  def getCmdLineArguments(args: Array[String]): CmdConfig = {
```

Comment:
Could you add tests for this like [this](https://github.com/AbsaOSS/enceladus/blob/master/standardization/src/test/scala/za/co/absa/enceladus/ConfigSuite.scala#L117)?